.gitignore

```diff
@@ -1,11 +1,9 @@
-bin/
-gen/
-obj/
-build/
-app/release/
+*.iml
 .gradle/
-.idea
-app/.cxx
-app/.externalNativeBuild
 local.properties
-*.hprof
```

Comment:
I think I had added this line because the file was pushed after I tried profiling. Is there any reason for deleting this line? (.hprof)

Reply:
Good point, I'll add it back.